### ISSUE TITLE
Coroutine in RshkAds.cs never called and package download link not pointing to right file.

### ISSUE THREAD

**Issue by Onoa** [viccorvalan/Unity5-Admob-with-Mediation#1]
Hi,

I am using your plugin (thank you btw, it's very useful) and I have noticed that this code never gets called when debugging:

Instance.StartCoroutine(RewardedDone());

// We need this to avoid errors on Android

IEnumerator RewardedDone()

{

yield return new WaitForSeconds(1f);

AudioListener.pause = false;

OnRewardedCompleted();

RequestRewarded();

}

In fact, it looks like any call to a function that yield (Coroutine) will not be executed if called from any of the delegates in your example script.

Could you please confirm that you also aren’t able to execute this code as well? 

Also, the link on the main page (https://goo.gl/HnYYOQ) to download the latest package points to the wrong folder: heyzapMediation-101.unitypackage

**Comment by ayublin** [comment]
@pescode I second this, can you please correct the package link (https://goo.gl/HnYYOQ)? I have tried to download it and it's actually a wrong one. Thanks in advance, this is such a time saver!

**Comment by pescode** [comment]
@Onoa 
Yes... that's something strange... I have a problem on Android when I was using that call as a function and not as a Coroutine... I'm looking at my code and now I'm using as a function again (at least on iOS)

@ayublin 
Sorry!... the correct file is now available
